polkadot/xcm/src/v3/mod.rs

```diff
@@ -1576,118 +1576,6 @@ impl<Call> TryFrom<OldInstruction<Call>> for Instruction<Call> {
 #[cfg(test)]
 mod tests {
 	use super::{prelude::*, *};
-	use crate::v2::{
```

Comment:
I would maybe remove these tests really when removing the whole v2, can we use `#[allow(deprecated)]` for tests?

Reply:
I removed the test in the spirit of "we don't guarantee anything with deprecated versions". The code is still there, but we already don't want the maintenance burden of making sure these tests pass. Although we won't push new changes to the version, so they should always pass.

That was my reasoning, but I can just allow deprecated.